grr/server/grr_response_server/flows/general/osquery_test.py

```diff
@@ -152,15 +150,12 @@ def testFailure(self):
       self._RunQuery("UPDATE time SET day = -1;")
 
 
-def _CreateDictTable(column_number, row_number) -> List[OrderedDict]:
-  def CellValue(col_number, row_number):
-    return (f"col-{col_number}", f"col-{col_number}, row-{row_number}")
-
-  return [
-    OrderedDict(
-      CellValue(col_number, row_number) for col_number in range(column_number)
-    ) for row_number in range(row_number)
-  ]
+@contextmanager
+def PatchTruncationLimitTo(new_value: int):
+  old_value = osquery_flow.TRUNCATED_ROW_COUNT
+  osquery_flow.TRUNCATED_ROW_COUNT = new_value
+  yield
+  osquery_flow.TRUNCATED_ROW_COUNT = old_value
```

Comment:
No need for this helper, `mock.patch.object` has this functionality (e.g. see [here][1]).

[1]: https://github.com/google/grr/blob/fe2563069b0a8c6fdea3fe24a3c938c16391a5e3/grr/server/grr_response_server/artifact_test.py#L517-L518